### ISSUE TITLE
Hide lock icon if OTR is inactive

### ISSUE THREAD

**Issue by licaon-kter** [redsolution/xabber-android#858]
Thanks

2.6(557)

**Comment by andrewnenakhov** [comment]
But lock icon is used to enable OTR. If we hide it, users will be unable to enable it. Right?

**Comment by licaon-kter** [comment]
No, when inactive is set in Setttings, all OTR options are deactivated, grey, useless.

**Comment by andrewnenakhov** [comment]
Hm. indeed. Why would anyone choose disabled OTR mode? We've got too many options in settings, gotta shrink their numbers.

**Comment by licaon-kter** [comment]
You know that OTR has it's drawbacks (no multi device), I'd happily advise users to disable it so I don't end up with `?OTR:AAMDJ+MVmSfjFZcAAAAAAQAAAAIAAADA1g5IjD1ZGLDVQEyCgCyn9hb` messages on screen.

**Comment by andrewnenakhov** [comment]
@licaon-kter that won't help anyway if remote party will send you encrypted messages that you can't decrypt.

**Comment by licaon-kter** [comment]
No, I said that I advise them to "disable" it so we don't have mistakes. ;)
